packages/gestalt/src/ButtonToggle.tsx

```diff
@@ -274,8 +301,24 @@ const ButtonToggleWithForwardRef = forwardRef<HTMLButtonElement, Props>(function
       onFocus={(event) => {
         onFocus?.({ event });
       }}
-      onMouseDown={handleMouseDown}
-      onMouseUp={handleMouseUp}
+      onKeyDown={(e) => {
```

Comment:
My only feedback is that this seems repeated, but as I thought about, I'm not sure if there is a better way to abstract this. Can you pass in the 'experiment' name or the flag, to the hook itself. 

e.g. in `useButtonToggleAnimation` you might be able to do the if statement there instead, so in the future for more VR animation experiments, we can do all of that in that one method

Reply:
@AlbertCarreras What's your take on this? I just wanted to have the experiment related code to be in one file. So that when we eventually remove the experiment code, we only change one file instead of multiple.

Reply:
it's stylistic. it's not super dry as is, but it's very explicit in the component itself and less opaque. 

Technically this animations are VR only so we can gate outside.

If we gate inside, then we'll reuse it in more components and it will be more dry. 

It doesnt bother me either way. Not opinionated.